SofaKernel/modules/SofaBaseMechanics/MechanicalObject.inl

```diff
@@ -2623,6 +2623,32 @@ void MechanicalObject<DataTypes>::getConstraintJacobian(const core::ExecParams*
 }
 
 template <class DataTypes>
+void MechanicalObject<DataTypes>::buildIdentityBlocksInJacobian(const sofa::helper::vector<unsigned int>& list_n, core::MatrixDerivId &mID)
```

Comment:
We can remove Jacobians from Mechanical States and store them in an other component. 
But this would be an other modification of SOFA core.